src/components/calcite-alert/calcite-alert.scss

```diff
@@ -1,147 +1,130 @@
-    @include font-size(-1);
+    @apply text--1h;
   }
   @include slotted("alert-message", "*") {
-    @include font-size(-2);
+    @apply text--2h;
   }
   ::slotted(calcite-link) {
-    @include font-size(-2);
+    @apply text--2h;
   }
 }
 
 :host([scale="l"]) {
-  --calcite-alert-width: 60em;
-  --calcite-alert-spacing-token-small: #{$baseline/1.25};
-  --calcite-alert-spacing-token-large: #{$baseline * 1.25};
+  --calcite-alert-width: theme("maxWidth.4xl");
+  --calcite-alert-spacing-token-small: theme("spacing.5");
+  --calcite-alert-spacing-token-large: theme("spacing.8");
   @include slotted("alert-title", "*") {
-    @include font-size(0);
+    @apply text-0h;
   }
   @include slotted("alert-message", "*") {
-    @include font-size(-1);
+    @apply text--1h;
   }
   ::slotted(calcite-link) {
-    @include font-size(-1);
+    @apply text--1h;
   }
 }
 
 :host {
-  @apply shadow-2;
-  display: flex;
-  position: fixed;
-  justify-content: center;
-  align-items: center;
-  pointer-events: none;
+  @apply flex
+    fixed
+    justify-center
+    items-center
+    pointer-events-none
+    bg-foreground-1
+    rounded
+    opacity-0
+    inset-x-0
+    bottom-0
+    shadow-2;
   margin: 0 auto;
   width: var(--calcite-alert-width);
-  max-width: 90%;
-  max-height: 0;
-  background-color: var(--calcite-ui-foreground-1);
-  border-radius: var(--calcite-border-radius);
-  opacity: 0;
-  left: 0;
-  right: 0;
-  bottom: 0;
-  pointer-events: none;
-  z-index: 101;
-  transform: translate3d(0, $baseline, 0);
-  transition: 300ms $easing-function, opacity 300ms $easing-function, all 0.15s ease-in-out;
-  border-top: 0px solid transparent;
+  max-width: 90%; // TODO
+  max-height: 0; // TODO - wat
+  z-index: 101; // TODO - z-50?
+  transform: translate3d(0, $baseline, 0); // TODO
+  transition: 300ms $easing-function, opacity 300ms $easing-function, all 0.15s ease-in-out; // TODO
+  border-top: 0px solid transparent; // TODO
   @media only screen and (max-width: $viewport-medium) {
-    width: 100%;
-    max-width: 100%;
-    border-radius: var(--calcite-border-radius) var(--calcite-border-radius) 0 0;
+    @apply w-full max-w-full rounded-t;
   }
 }
 
 // focus styles
 .alert-close {
-  @include focus-style-base();
+  @apply focus-base;
   &:focus {
-    @include focus-style-inset();
+    @apply focus-inset;
   }
 }
 
 :host([active]:not([queued])) {
-  opacity: 1;
-  max-height: 100%;
-  transform: translate3d(0, -$baseline, 0);
-  pointer-events: initial;
-  border-top-width: 3px;
+  @apply max-h-full opacity-0 pointer-events-auto border-t-2;
+  transform: translate3d(0, -$baseline, 0); // TODO jrowlingson - transform-gpu
 
   @media only screen and (max-width: $viewport-medium) {
     transform: translate3d(0, 0, 0);
   }
 }
 
 @include slotted("alert-title", "*") {
-  @include font-size(0);
-  color: var(--calcite-ui-text-1);
-  margin: 0;
-  font-weight: 500;
+  @apply text-0h text-color-1 m-0 font-medium;
```

Comment:
line 109 before change it's a CSS var, making this change `text-color-1` keep the CSS var?